event_repository.js

```diff
@@ -14,7 +14,7 @@ function getAllEvents() {
 
 function getEventById(id) {
     return new Promise((resolve, reject) => {
-        const filteredEvent = events.filter((item) => {
+        const filteredEvent = events.find((item) => {
             return item.id === id;
         })[0];
```

Comment:
According to MDN, find "returns the value of the first element in the array that satisfies the provided testing function", so we must get rid of [0] here.

I will change this and merge the PR to master :)